qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-validate.yaml

```diff
@@ -0,0 +1,28 @@
+name: "QHub Config Validate - Pull Request"
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "qhub-config.yaml"
+
+{% raw %}
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+{% endraw %}
+
+jobs:
+  qhub-validate:
+    name: 'Qhub'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Infastructure'
+        uses: actions/checkout@master
+      - name: 'Qhub Format'
+        uses: Quansight/qhub-github-actions@v0.1-beta
+        with:
+          qhub_actions_version: 0.0.1
```

Comment:
Does this allow us to use the qhub version 0.1.19 for validation?

Reply:
The version field doesn't do anything right now. I have kept it for a later stage when the qhub-bot can do other tasks as well. Should I remove version field ?